src/core.jl

```diff
@@ -1,16 +1,18 @@
 export DSSCore
 
 "Module with the low-level API for OpenDSSDirect"
-module DSSCore 
+module DSSCore
 
 if is_windows()
     if Int == Int64
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win64\\OpenDSSDirect.DLL"
     else
         const dsslib = "$(dirname(@__FILE__))\\..\\deps\\win32\\OpenDSSDirect.DLL"
     end
-else
-    const dsslib = "$(dirname(@__FILE__))/../deps/libopendssdirect.so"
+elseif is_linux()
```

Comment:
@tkelman, that's why we need a BinDeps2! 😈  I cringe at including binary libraries as it is.

I'm trying out a fix for this on Travis now. That still doesn't help for nonstandard platforms where someone may compile the library themselves. But, hopefully, we'll improve our library build process.